backend/src/org/commcare/suite/model/Detail.java

```diff
@@ -70,31 +84,13 @@ public Detail() {
     }
 
     public Detail(String id, DisplayUnit title, String nodeset,
```

Comment:
refactored this to only use 1 constructor because nothing was calling the other 2